### ISSUE TITLE
Add option to output log to console (for situations when there's no access to log files)

### ISSUE THREAD

**Issue by ayeltsov** [Azure/azure-storage-azcopy#2516]
### Which version of the AzCopy was used? 
10.22.1

### Which platform are you using? (ex: Windows, Mac, Linux)
Windows
### What command did you run?
n/a

I'm working with automation on the system that is does not allow any connections to it. No RDP, no network access, nothing.

The only way to see what's going on is console output of the runner.

For troubleshooting, I'm collecting some logs with azcopy. 
But when azcopy runs into a problem, it does not display enough information on the console, only in the log. And I can't get to the log files (that's the whole reason I'm trying to run azcopy). 

Please add option to output logs to console instead of, or in addition to, log files.

**Comment by vibhansa-msft** [comment]
AzCopy is very verbose in logging and hence we prefer to dump logs to file. Dumping it on console will hang up your console depending on the job size and most customers working with automation do not like that part. Once your job is complete you can use the job-id to get the log file and analyze it.
" when azcopy runs into a problem, it does not display enough information on the console" can you point something specific here which can help us improve our logging.

**Comment by ayeltsov** [comment]
> AzCopy is very verbose in logging and hence we prefer to dump logs to file. Dumping it on console will hang up your console depending on the job size and most customers working with automation do not like that part. 

That's fine, but you're answering as if I asked to make it default for everyone. I did not. The answer does not apply to what I'm asking. Also I don't get "hang up console" part, as far as I know that is not a thing. Again, let me make the choice.

> Once your job is complete you can use the job-id to get the log file and analyze it. 

Please take another look at the scenario, that is not possible - I have no interactive access to the machine. I can launch something, wait for 1 hour, observe console output.

> " when azcopy runs into a problem, it does not display enough information on the console" can you point something specific here which can help us improve our logging.

This is not the correct mindset. I need to be able to troubleshoot any problem by looking at just console output. Which means I cannot be specific by definition - I need to see everything that may indicate root cause of the problem. In on particular narrow scenario, some of the files were in use. So would need to see that. But that's just one example.

**Comment by vibhansa-msft** [comment]
Closing this here, we will add this item to our backlog but as of now this will be on low priority. If you are blocked on this one, you can fork the repo and make the required changes. Feel free to submit back the PR as a contribution to AzCopy.
